starknet_py/net/http_client.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, url, session: Optional[aiohttp.ClientSession] = None):
     def http_session(self) -> ClientSession:
         if self.session is not None:
             # noinspection PyTypeChecker
-            return nullcontext(self.session)
+            return nullcontext(self.session)  # pyright: ignore
```

Comment:
We should add a issue about covering this line with tests

Reply:
https://github.com/software-mansion/starknet.py/issues/391